open-metadata-resources/open-metadata-deployment/docker/egeria-uistatic/dist/nginx.conf

```diff
@@ -12,4 +12,13 @@ server {
     location / {
         try_files $uri /index.html;
     }
+
+    # Revese proxy all the API call to ui-chassis-spring hosted Web API
+    location /api {
+        proxy_pass https://localhost:8443/api;
```

Comment:
This effectively needs to be a variable to match the helm chart - and even if using plain docker, localhost will be insufficient.

Scripted using env vars is one option, but I think the best (at least in k8s) is to use a k8s volume perhaps from a configmap.

Will defer this thought until after release ship .